src/go/rpk/pkg/utils/files.go

```diff
@@ -83,28 +83,28 @@ func WriteBytes(fs afero.Fs, bs []byte, path string) (int, error) {
 	return len(bs), afero.WriteFile(fs, path, bs, 0o600)
 }
 
-func FileMd5(fs afero.Fs, filePath string) (string, error) {
```

Comment:
this is the only golang exported function that changed name. changed from `FileMd5` to `FilenameHash`. from [search](https://github.com/search?q=org%3Aredpanda-data%20FileMd5&type=code) in redpanda-data org, doesn't look like it is used anywhere.